app/Login/Login.tsx

```diff
@@ -0,0 +1,143 @@
+"use client";
+
+import { useForm } from "react-hook-form";
+import { useEffect, useRef, useState } from "react";
+import classNames from "classnames";
+import { FaEye, FaEyeSlash } from "react-icons/fa";
+import styles from "../Login/Login.module.scss";
+import {
+  PLACEHOLDEREMAILLOGIN_OBJECT,
+  PLACEHOLDERPASSLOGIN_OBJECT,
+} from "@/public/script";
+
+interface LoginForm {
+  email: string;
+  password: string;
+  rememberMe: boolean;
+}
+
+export const Login = () => {
+  useEffect(() => {
+    document.title = "Log in - Chakrulos";
+  }, []);
+
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<LoginForm>();
+
+  const [showPassword, setShowPassword] = useState(false);
+
+  const onLoginFinished = (values: LoginForm) => {
+    console.log("FORM SUBMITTED", values);
+
+    fetch("/api/login", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(values),
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        if (data.success) {
+          // Warmatebit Daloginda
+        } else {
+          // Ver gaira login
+        }
+      })
+      .catch((error) => {
+        console.error("Error:", error);
```

Comment:
why console let it be alert

Reply:
this is backend code i can delete them, but i think we need this

Reply:
this is your code not backend user can not see error in console let it be alert